main/end-to-end-tests/cypress/integration/workspaces.spec.js

```diff
@@ -102,4 +145,39 @@ describe('Launch a new sagemaker workspace', () => {
       .parent()
       .contains('PENDING', { timeout: 600000 });
   };
+
+  const checkInstanceTypeInSelectionCard = workspaceParam => {
+    navigateToConfigurationCard(workspaceParam);
+
+    cy.get('[data-testid=configuration-card]').contains('Instance Type');
+  };
+
+  const navigateToConfigurationCard = workspaceParam => {
+    navigateToWorkspaces();
+
+    cy.get('[data-testid=workspaces]');
+
+    // Click create new workspace button
+    cy.get('button[data-testid=create-workspace]').click({ force: true });
+
+    // Select the type of environment you want to launch
+    cy.get('[data-testid=env-type-card]')
+      .contains(workspaceParam.workspaceTypeName)
+      .click();
+
+    // Click next
+    cy.get('button')
+      .contains('Next')
+      .click();
```

Comment:
Nitpick: I think we can just put the content of this method in the method `checkInstanceTypeInSelectionCard` since we're not calling method `navigateToConfigurationCard` from anywhere else.

Reply:
Got it! I thought modular was better in case it had other uses, but you're right that it doesn't so I will combine them.